src/reducer.js

```diff
@@ -118,6 +128,20 @@ export const actionReducers = {
   },
 
   /**
+   * Update the bunsen model
+   * @param {State} state - state to update
+   * @param {Action} action - action
+   * @returns {State} - updated state
+   */
+  [CHANGE_VIEW]: function (state, action) {
```

Comment:
Not to get confused with Ember but it is actually called a computed property name.

> Starting with ECMAScript 2015, the object initializer syntax also supports computed property names. That allows you to put an expression in brackets [], that will be computed as the property name.

[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)